inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/config/RemoteConfigManager.java

```diff
@@ -205,6 +205,12 @@ private boolean reloadDataProxyConfig(String clusterName, String clusterTag, Str
             LOGGER.info("start to request {} to get config info with params {}", url, request);
             CloseableHttpResponse response = httpClient.execute(httpPost);
             String returnStr = EntityUtils.toString(response.getEntity());
+
+            if (response.getStatusLine().getStatusCode() != 200) {
```

Comment:
Will the `response.getStatusLine()` be null?

Reply:
<img width="981" alt="image" src="https://user-images.githubusercontent.com/45282474/229756142-71345078-3034-4d0b-81e7-efb84ef11e4b.png">

The constuctor of HttpResponse requires statusline not null.